tests/conftest.py

```diff
@@ -0,0 +1,23 @@
+import logging
+from pathlib import Path
+
+import pytest
+
+from command_line_assistant import config, logger
+
+
+@pytest.fixture(autouse=True)
+def setup_logger(request, tmpdir):
```

Comment:
It's [preferable](https://docs.pytest.org/en/stable/how-to/tmp_path.html#temporary-directory-location-and-retention) to use `tmp_path` instead of `tmpdir` these days. That'll also save some work later.